packages/app/src/app/pages/Pro/index.tsx

```diff
@@ -99,6 +103,8 @@ const ProPage: React.FC = () => {
     return null;
   };
 
+  if (location.search.includes('v=2')) return <ProPageV2 />;
```

Comment:
jump out if `v=2` is passed, this way we can test it standalone and link from dashboard and homepage